packages/sdk-components-react/src/bold.ws.tsx

```diff
@@ -4,9 +4,9 @@ import {
   type PresetStyle,
   type WsComponentMeta,
   type WsComponentPropsMeta,
-} from "./component-meta";
+} from "@webstudio-is/react-sdk";
+import { b } from "@webstudio-is/react-sdk/css-normalize";
```

Comment:
isn't it going to treeshake if you just import it without a separate entry point?

Reply:
Semantically not good to export tag styles from sdk without namespace. Though namespace export or object will kill treeshaking.